packages/fx-core/tests/plugins/resource/util.ts

```diff
@@ -0,0 +1,32 @@
+import { compileHandlebarsTemplateString } from "../../../src";
+import { ScaffoldArmTemplateResult } from "../../../src/common/armInterface";
+
+export function mockSolutionUpdateArmTemplates(
+  mockedData: any,
+  template: ScaffoldArmTemplateResult
+): ScaffoldArmTemplateResult {
+  return {
+    Modules: template.Modules,
+    Orchestration: {
+      ParameterTemplate: {
+        Content: compileHandlebarsTemplateString(
+          template.Orchestration.ParameterTemplate!.Content,
```

Comment:
maybe need to handle cases that resource plugin doesn't provide this content? We can also update this logic if we meet this case in the future.

Reply:
Nice catch. Fixed.